### DISCUSSION TITLE
Defining minimim fragment distance for MCES

### DISCUSSION THREAD

**Discussion by mwojcikowski** [rdkit/rdkit#7974] [Development]
I would found it useful to have minFragSeparation as maxFragSeparation is already a parameter. I Found cases where rings were fragmented into adjacent bits (e.g. N migrated in pyridine). Before I give it try I'd like to ask @DavidACosgrove if he had some experiments with such threshold, as there seem to be `minFragSeparation` function implemented without being applied anywhere.

**Comment by DavidACosgrove** [comment]
@mwojcikowski, that's good digging.  I honestly can't remember whether that was an idea that I failed to follow through on or something where I tried it, anddidn't like the results but left the code in just in case it might be useful later.  Can you provide a couple of simple examples of the types of molecule pairs you're thinking of and what results you'd like, and I'll take a look?
